docs/variability.md

```diff
@@ -91,14 +91,15 @@ If your machine has really limited resources or creating a clean environment has
 
 When creating your thresholds for failure, either mental or programmatic, use aggregate values like the median, 90th percentile, or even min/max instead of single test results.
 
-The median Lighthouse score of 5 runs is twice as stable as 1 run, and tools like [lighthouse-ci](https://github.com/GoogleChrome/lighthouse-ci/) can run Lighthouse multiple times for you automatically.
+The median Lighthouse score of 5 runs is twice as stable as 1 run. There are multiple ways to get a Lighthouse report, but the simplest way to run Lighthouse multiple times and also get a median run is to use [lighthouse-ci](https://github.com/GoogleChrome/lighthouse-ci/).
 
 ```bash
-# Note: you must be in a git repository with at least one commit for this to work.
 npx -p @lhci/cli lhci collect --url https://example.com -n 5
 npx -p @lhci/cli lhci upload --target filesystem --outputDir ./path/to/dump/reports
 ```
 
+> Note: you muse have [Node](https://nodejs.org/en/download/package-manager/) and [npx](https://www.npmjs.com/package/npx#install) installed.
```

Comment:
```suggestion
> Note: you must have [Node](https://nodejs.org/en/download/package-manager/) and [npx](https://www.npmjs.com/package/npx#install) installed.
```

doesn't node just come with `npx` these days? I didn't realize it needed to be a separate install

Reply:
node coming with npx? never heard of search a thing. isn't npx a npm package?

now that I've finally looked it up, it does seem the node ppl maintain `npx`. and `which npx` shows I'm using it. ok. will revise